### PR TITLE
lube now requires 10u to apply to a tile and has an extremely more obvious sprite (blue sparkles) and no longer bypasses noslips

### PR DESCRIPTION
**Pull request by k-k-karma**
:cl: karma
balance: lube now requires 10u to apply to a tile and has an extremely more obvious sprite (blue sparkles) and no longer bypasses noslips
/:cl:

[why]: fuck you gregg

### CODE REVIEW

**Comment by G-erma** [comment]
saltcode, just dont walk on it idiot boy

**Comment by k-k-karma** [comment]
nice try gregg

**Comment by k-k-karma** [comment]
just dont walk on it is a shit argument and you know it, it's nearly invisible and hard to avoid mid-combat and not walking on it is half the use of it, you can selectively block areas off so people have to go an extremely longer way around to catch you if they're chasing or be stunned

**Comment by TrickyDickRichard** [comment]
who lubed you

**Comment by k-k-karma** [comment]
john mclaime and some guy named gregg

**Comment by TrickyDickRichard** [comment]
To bad

**Comment by k-k-karma** [comment]
thanks for the +1

**Comment by Kawaii-Big-Boss** [comment]
NOTE: THIS IS GRUDGECODE
Because this is grudgecode, I'm inclined to give it an instant -1 just because of it. There are better ways to nerf lube than just "making it not stun", don't be Spacedong or Robustin, actually try to make items not suck.
You even mention how it's nearly invisible, and how you can block people off. Why not, I dunno, change that part?
Obviously because a player named Gregg (TDCoolguy? what?) slipped you and you died, so now you're mad and you're grudgecoding it. Because if there's anything Hippie needs, it's a third saltcoder.

**Comment by danhibiki** [comment]
No, there would be little reason to use lube over water beyond making them slip a couple more tiles

**Comment by Jujumatic** [comment]
lol just get gud. or alternatively use lube yourself. or a banana. or soap. come on dude

**Comment by Carbonhell** [comment]
Lol grudgecode
either nerf it properly or don't, don't make it useless

**Comment by k-k-karma** [comment]
Kawaii-Big-Boss, please note the grudgecode aspect is a meme and everyone on hippie agrees that lube is apart of the meta for just about everything. You always use it when you're a sling due to the fact it's the best stun weapon.
"how you can block people off. Why not, I dunno, change that part?" - Did you read the PR? it makes it so it stuns while you're slipping on it. That part is changed. Lube will actually now require skill and creativity to use it effectively instead of spray and win.

**Comment by k-k-karma** [comment]
grudgecode aspect is a meme - lube has been on everyone's mind. it is apart of THE meta otherwise john mclaime wouldnt use lube smoke bombs, kbb wouldnt use it as a sling because they have nogun and it's the best next alternative.

however, i've changed it so you are forced to crawl for 15 seconds instead of stunned for 15 seconds

**Comment by Kawaii-Big-Boss** [comment]
@k-k-karma The issue I have is that this is outright the laziest way to nerf lube, and more or less makes it worthless.
You're not considering why lube is so good compared to water, and maybe looking at balancing out the scales. You're instead only interested in bringing lube down, which just makes it worthless.
I don't only use lube when I'm sling, I use it all the time. It's godtier and absolutely could use a nerf. But not like this. The last thing this server needs is /tg/-tier "I DED" PRs which nerf an item to complete uselessness under the guise of it "nerfing an item that's overpowered."
Knockdown doesn't make it totally worthless but in doing so you fail to accomplish your original goal of making it not meta. It's still an amazing deterrent that you can't walk around or do anything, and it's still gonna make you drop your items which is what I ACTUALLY use lube for, not to get the HoS but rather to get the HoS' gun. It's also still incredible in maint halls which are only one tile long, meaning that spraying lube down there is still impenetrable, whereas every other means of slips have some way around them.

**Comment by k-k-karma** [comment]
did you re-read the PR after i updated it? this nerf seems reasonable to me

**Comment by yoyobatty** [comment]
The crawl code is nerfed via upstream pr.

**Comment by k-k-karma** [comment]
@Kawaii-Big-Boss the easiest way to nerf it would be to remove the bypass on noslips but someone has already tried that. if this isn't good enough give me some ideas

**Comment by k-k-karma** [comment]
also besides that Knockdown doesnt make u drop ur items sorry sweaty

File: code/game/turfs/open.dm [hunk 1]
```diff
@@ -240,7 +240,10 @@
 		var/olddir = C.dir
 		C.moving_diagonally = 0 //If this was part of diagonal move slipping will stop it.
 		if(!(lube & SLIDE_ICE))
-			C.Paralyze(knockdown_amount)
+			if (lube == (GALOSHES_DONT_HELP|SLIDE))/*hippie start - nerfs lube so you only crawl instead of fullstun.*/
```
Comment (yoyobatty) [inline code/game/turfs/open.dm]: `CHECK_MULTIPLE_BITFIELDS(lube, GALOSHES_DONT_HELP|SLIDE)`

**Comment by yoyobatty** [comment]
Needs headmin input.

File: code/game/turfs/open.dm (hunk 1 above)
Comment (k-k-karma) [reply code/game/turfs/open.dm]: done

**Comment by Kawaii-Big-Boss** [comment]
@k-k-karma The way I would nerf it is simple, attempt to reverse the properties between Water and Lube, or equalize them.
Reversing would mean that water goes 3 tiles for 5u, lube 1 tile for 10u. It also doesn't last as long. A concern with doing this is that lube is still a great deterrent that has no means around it outside of doing cheeky floortile stunts, which already aren't going to get you very far. A second concern would be Water Vapor and how it could potentially become even more annoying.

Equalizing would mean that both water and lube can be walked over, both 3 tiles for 5u (or both 1 tile for 10u). This doesn't change that lube is a straight upgrade to water, so perhaps something else could be thrown on. I already don't like how lube and water aren't very distinguishable in terms of appearance, and how powerful it is in maintenance as a result, so I could suggest adding a sparkling effect to it that makes it far more visible. Still good for slipping people who are chasing you, still a very long stun, but also very easy to see coming and isn't godtier deterrents. Overall, a bit more discussion could be done about how lube could be nerfed, I myself have wanted to nerf it but despite that lube is not very high on my list, thanks to other issues existing in the game that are even more of a concern than lube is.

Your way of doing it simply makes lube statistically worse, which is /tg/-tier and while sometimes justified, tends to ruin items and make them totally worthless compared to their counterparts while also sterilizing combat and ensuring things like Tasers and Ebow stay at the top tiers until they get nerfed as well, causing everything to slow down while no alternatives to stun combat are ever properly buffed to be a match against stuns (i.e swing combat to make melee weapons not completely useless and inferior to stunbatons at all times). What this PR would do is make lube worse, water still sucks, and soap can still be picked up or walked over with no-slips, instead of addressing the problems in the entirety of the balance. So why even use lube then? You didn't fix how it can't be walked past, but you instead made it that it only knockdowns instead of stunning, which albeit still makes it useful due to people being slowed, but because they don't drop their weapons they can still use shields and spam shots down the hallway. As a result you're buffing the exact types of players you just complained about in another PR.

**Comment by Jujumatic** [comment]
i just want to lube the hallways and piss off half the station crew 4no raisin

it gives you some good experience for when you will inevitably get catbanned

**Comment by danhibiki** [comment]
Best way to combat lube is to git gud

**Comment by k-k-karma** [comment]
so you want bypasses noslip removed from lube if less than 3u units are used on a tile?
how many units does a fire extinguisher put on a tile?

**Comment by Kawaii-Big-Boss** [comment]
No, karma, that's not what I said.
Also fire extinguishers being filled with chemicals was a mistake. It's nothing but trash.

**Comment by k-k-karma** [comment]
yeah

**Comment by k-k-karma** [comment]
honestly though this doesn't make lube worthless, it's better than water and soap and all other slips still. people are forced to crawl for 15s making them an easy target for disarms, sling glare, etc. i don't see why you think it's absolute dog shit tier after this.

**Comment by Jujumatic** [comment]
fill a fire extinguisher with unstable mutagen and spray it at people. look at the fun that happens

**Comment by danhibiki** [comment]
It most certainly isn’t better than a banana, which is easy to get a 100 potency

**Comment by Kawaii-Big-Boss** [comment]
@k-k-karma The problem I have with just making it statistically worse is just that. You're making it worse. That's all you're doing. Statistic nerfs are something that /tg/ pulls all the time and every time it ends up happening, there's always a revolt in OOC over how fucking stupid it is.
I proposed my changes as a means of adding options instead of outright nerfing lube, and ensuring every single slip isn't a guaranteed deterrent. If you suck, you'll still slip on lube and you'll still die. It just means that you can't lay down lube in a maint hallway and sit there for five years waiting for someone to slip, which actually was an old strategy used by powergamers back in 2015-2016. People will see it coming and can walk past it. Bananas and soap can be picked up if players see it coming, it's why you're always cautious around clowns. Water... uhh, sucks, but if you were to buff it as I proposed, then it can become somewhat of a stealthier version of lube. Harder to see, less of a stun, but still viable unlike how it is in its current state where it's only really good for panic use with fire extinguishers.
On top of that, no-slip shoes, galoshes and the like become a universal counter to slips entirely, therein making traitor have an advantage. I don't know why you'd want to buff traitor more, but if you dislike how "sec has constantly been buffed" then this is the way to go, rather than just making lube suck.

**Comment by k-k-karma** [comment]
of course traitors are going to have advantage they're antagonists

**Comment by danhibiki** [comment]
One on one the traitor should always have an upper hand, it’s suppose to take some sort of cooperation between the station to mitigate their success. It’s the entire concept of the game

**Comment by k-k-karma** [comment]
and of course i'm making it worse that's the definition of a nerf. you go fucking nerf lube yourself if you dont like it dude but other than that this seems reasonable to me

**Comment by danhibiki** [comment]
The reason bananas are good also is because the only indicator there is a banana is a visual cue, there is no audio like with lube deployment

**Comment by k-k-karma** [comment]
this is not a banana nerf

**Comment by danhibiki** [comment]
My point is lube isn’t nearly as strong as your making it out

**Comment by k-k-karma** [comment]
it really fucking is, the best players use it constantly cos it's fucking god tier

**Comment by k-k-karma** [comment]
you can coat a 3x3 area in lube instantly with a fire extinguisher

**Comment by Kawaii-Big-Boss** [comment]
So instead of responding to my post, you just say _lol you do it instead LOL_
No, you're the one who decided to step up to nerf lube, you do it right instead of just making it complete shit. You're also not bothering to address anything I'm saying and you're instead saying "WTF TRAITOR SHOULD BE GOOD", which is the exact point I was making if you want that to be the case.
This is why I hate grudgecode.
Also, what Dan is saying is also true. 100 potency bananas and the like are quite powerful and can be concealed somewhat. Do note that they do take more time to make than lube, which is why I wouldn't say they're quite on par, especially because no-slips still don't go through lube.

**Comment by danhibiki** [comment]
Rip up tiles, use ranged weaponry as a counter, don’t walk over it

**Comment by k-k-karma** [comment]
ripping up tiles takes time and if you fuck it up you're stunned for 15s, ranged weaponry doesn't matter because they're using it to run away from you instead of waiting you to slip over it for free stun

**Comment by k-k-karma** [comment]
@Kawaii-Big-Boss give me the simplest way to nerf lube that doesnt trigger your autism

**Comment by Kawaii-Big-Boss** [comment]
I already did.

**Comment by k-k-karma** [comment]
explain it simply i have autism

**Comment by danhibiki** [comment]
Ranged attacks like lasers and disablers go beyond your vision

**Comment by k-k-karma** [comment]
you do realize they can avoid them with that much of a range advantage

**Comment by danhibiki** [comment]
Almost like you could avoid lube unless your hugging someone’s ass

**Comment by Kawaii-Big-Boss** [comment]
Make lube glow like glowshrooms, or have a sparkling effect, or both. Remove GALOSHES_DONT_HELP, and replace it with NOSLIP_IF_WALKING or whatever it's called.
Keep the long ass stun.
Now, decide whether or not lube should be 10u for one tile (talking spray bottles, here) just like water. What this will do is keep people from using lube in fire extinguishers, but it's also gay and makes bananas the better choice instead.
Or you could make water 5u for 3 tiles (talking spray bottles, again) to be just like lube. I'd prefer this outcome, but beware of extinguishers.

**Comment by k-k-karma** [comment]
fine

**Comment by k-k-karma** [comment]
@Kawaii-Big-Boss done

**Comment by k-k-karma** [comment]
@Kawaii-Big-Boss https://i.imgur.com/jfgEmuC.gifv

**Comment by danhibiki** [comment]
You want that to be a floor overlay, Incase you didn’t consider that. Not all station tiles much less walkable tiles look like that

**Comment by zamolxius** [comment]
hurr durr im ded. also why blue sparkles wtf, i'd say more like a slight cyan tint AT MOST. also tbh 10u per spray would be kinda gay, but sure

**Comment by k-k-karma** [comment]
that is a floor overlay afaik and blue sparkles is for telegraphing as lube is far too powerful if not telegraphed

**Comment by zamolxius** [comment]
"far too powerful" not that powerful, tbh. also make it so having a few tiles of lube in a row keeps slipping you up but doesn't add more stuns so you can do a nice highway to space in escape

**Comment by k-k-karma** [comment]
that's what the first PR was previously

**Comment by k-k-karma** [comment]
i cant please everyone

**Comment by Shdorsh** [comment]
https://www.youtube.com/watch?v=9hxkEQv5L-E

**Comment by WarRage333** [comment]
I agree with making it more discernible from water

**Comment by t-f-w** [comment]
lame ass grudgecode..... -1

**Comment by Kawaii-Big-Boss** [comment]
@t-f-w Oh god what are you doing
INB4 LUBE STAYS INSANELY GOOD BECAUSE HEADMINS WHO DON'T PLAY THE GAME CAN VOTE ON PRS

Y E S  THE DOWNFALL OF HIPPIE BEGINS

**Comment by Carbonhell** [comment]
We have testmerges now dont we? Could be a good candidate

**Comment by k-k-karma** [comment]
was just discussing this in #coder-copter with pyko, not sure if tgs4 has them but i'd love to

**Comment by Carbonhell** [comment]
Both have it but we couldn't use them on tgs3 due to weird bug. It should work now, pretty sure a PR was testmerged some days ago too by gregg.

**Comment by k-k-karma** [comment]
yeah it was the hotswap port bug fix by cyberboss right? ill see if gregg'll testmerge

**Comment by bubberducky2** [comment]
-1 obvious grudgecode
adding in freeon sparkle is lazy of you to do so buddy
could you consider the fact that nerfing something that's supossedly gamebreaking won't stop others from using water instead? they never even see it coming.

just like one of you said, just make it more obvious but not that obvious, i dont want lube that screams out HEY IM RIGHT HERE ON THIS FLOORTILE, DONT WALK ON ME!

sure maybe they'll fuckin use magboots and galoshes for slips but they'll be slow as fuck and they can't chase you afterwards.

**Comment by k-k-karma** [comment]
it's not that obvious, if you're not paying attention you'll still slip on it. it feels like you're only -1ing it so you dont have to deal with the removal of an OP item from your meta

**Comment by t-f-w** [comment]
just make it so you cant put chems in a fire extinguisher. like kbb said, that shit was a mistake.

**Comment by k-k-karma** [comment]
how about only lube doesnt work in a fire extinguisher?

**Comment by bubberducky2** [comment]
this has devolved into cancer

**Comment by k-k-karma** [comment]
as  per bubberducky's suggestion it's now using the old no2 sprite instead of freon

**Comment by bubberducky2** [comment]
good

**Comment by TDcoolguy300** [comment]
Test merging.

**Comment by Kawaii-Big-Boss** [comment]
So far so good. It's still very powerful, but seems like it could be manageable now.
Wish the sprite wasn't old no2 though.

**Comment by k-k-karma** [comment]
probably the best one until i get a codersprite or something

**Comment by bardofrage** [comment]
While i do think lube is op, i don't think this is the way to go about it. -1

**Comment by McBawbaggings** [comment]
by removing it being noslip piercing, watch as people use spacelube as antag without the potential to fuck themselves over despite needing an increased amount of volume to utilise it
sure, you have to spray a lot more on the 10u setting which doesn't go as far but now you have a 15 second stun where traitors are immune to it and only shit like galoshes and magboots are counters

**Comment by HideAndSeekLOGIC** [comment]
@McBawbaggings suggestion: make only magboots have lube protection. This would mean traitors would have incredibly obvious tells (syndie magboots) and decreased movement speed. Would this counteract the problem?

**Comment by k-k-karma** [comment]
@McBawbaggings i haven't seen anyone do it in the time that it was testmerged ( still is)

**Comment by McBawbaggings** [comment]
@HideAndSeekLOGIC People would just bum rush CE magboots or EVA magboots in general to deal with it. I dislike the idea that something as dangerous as spacelube has some particular item in limited quantities render it as a non threat. It just encourages more powergaming by having a way to self proof against lube while denying others a chance to defend against it.

@k-k-karma feel free to ping romathebest with the fact that spacelube is countered by noslips and watch how he would abuse it, he's a good benchmark for showing over broken mechanics

**Comment by k-k-karma** [comment]
the original concept of this nerf was originally roma's ideas on how to nerf i t

**Comment by k-k-karma** [comment]
there, noslips no longer work on lube but you can still walk over it

**Comment by Kawaii-Big-Boss** [comment]
@McBawbaggings you can still walk over it though?
it's just as abuseable as the clown PDA or water. don't see the purpose. But whatever.
@bardofrage I figured you'd -1 the PR. At least say why you'd prefer lube stays the way it is/don't like how lube is being nerfed here, person-who-never-plays-the-game.

**Comment by yoyobatty** [comment]
I remember trying to address this a while ago, I didn't put up as much of a fight as this mad lad.

**Comment by k-k-karma** [comment]
i saw

**Comment by Kawaii-Big-Boss** [comment]
@yoyobatty KBB'S _**FUCKING PISSED**_

**Comment by yoyobatty** [comment]
🤷‍♂️

**Comment by steamp0rt** [comment]
lol have my useless -1

**Comment by xTrainx** [comment]
This shit makes foam and smoke lube useless, ruining my botany pranks -1

**Comment by danhibiki** [comment]
if your fun is in botany is predicated on lube then you were ruining botany

**Comment by xTrainx** [comment]
I keep permanently freezing botany with with frost oil

**Comment by danhibiki** [comment]
ayyy thats my boi

**Comment by Eagle-Eyess** [comment]
Imo lube is fine. If you want to nerf it than my only suggestion would be just giving it a more noticeable sprite. Otherwise you might as well remove it

**Comment by k-k-karma** [comment]
have you ever played with KBB as a sling or with john mclaime in general

**Comment by Eagle-Eyess** [comment]
lube is avoidable. Unlike most other stuns lube only punishes people who walk over it. While having an almost unseeable sprite can be annoying it is still noticable. That is why the only change I feel is needed is a sprite change

**Comment by k-k-karma** [comment]
and if it's sprayed underneath you while you're walking or you're trapped because it can go into a 3x3 area and you dont have a crowbar?

**Comment by k-k-karma** [comment]
"j-j-just dont walk on it!" 
shit argument, its like saying "j-just dodge le tazer shots!"

**Comment by Eagle-Eyess** [comment]
Look I’m not going to continue arguing with stupidity. You don’t like my opinion? Tough shit. Others want to +1 this and over rule me, that’s fine no hair off my back.

**Comment by xTrainx** [comment]
Just rest lol

**Comment by k-k-karma** [comment]
thats a bug

**Comment by JohnMcLaime** [comment]
It needs a nerf, i would change it so its no longer "spam it till someone makes 1 mistake and they die". Lube doesnt only punish people who walk over it, you can lube around a person, they cant move, they die to a laser fire. Lube while someone is chasing you, now you get easy getaway and maybe a frag or two. It is so broken that i use it as anti stun measure, in form of voice activated smoke-lube grenade.
Oh but it can be "countered" by crowbarring a tile or putting a tile on floor. You are still forced to either cut the chase or waste time by equping crowbar in your active hand, that gives luber extra time to either lube more stuff or shoot you, same with tiles. It has barely has any disadvantages to the user, just a small storage slot for sprayer and 1 min to get into maint and smash the barricade to create it is all that it takes to get it. As long as you dont make a mistake of slipping on your own lube or cornering yourself with it, you will have huge tactical advantage.
What is a good reason to keep such weapon ingame? Has barely any downsides to the user, extremly easy to get, more spammable than whole armory in your bag of holding and if you victim makes 1 mistake, they get stunned long enough for you to cuff/kill them, even if they dont make that mistake, it still gives you advantage in positioning. Balance this shit, dont be a cuck like those other coders that just let it roam free for years.

**Comment by zamolxius** [comment]
just make it continuously apply motion in a direction while your on lube so you can make epic slides to escape airlock
make it disappear more slowly but not stun any more
if you hit a wall yoh might get stunned and damaged but thats it
this makes it more unique than lol upgraded water

**Comment by k-k-karma** [comment]
that was the original PR then i changed it

**Comment by JohnMcLaime** [comment]
Yes i suggested something like that before, should only keep you stunned as long as you are sliding, increase the lifetime of it in return, you still loose whatever you had in hand when you stepped on it, still useful but more balanced since it removes the death penaly of a stun.

**Comment by k-k-karma** [comment]
yeah that was the original pr but nobody liked it

**Comment by yobob591** [comment]
they are called fucking noslips why does lube slip you +1

**Pull request merged**